Added support for an iOSOpenDevPort setting

stonesam92 opened the pull request:
Allows users to specify the port on which to connect to for SSH/SCP like Theos' `THEOS_DEVICE_PORT` variable.

Primarily for the benefit of users tunnelling SSH through USB using [usbmuxd](http://www.iphonedevwiki.net/index.php?title=SSH_Over_USB).

kokoabim commented:
Before I merge this, will you change the use of $hostPort to be like $hostAddress (i.e. a local function variable from a passed parameter)?

stonesam92 commented:
I've added it as a function parameter to all the relevant function/action definitions which take an address parameter, and modified the usages texts to reflect the change.

Currently, I've added it as the last parameter for each function, in case any code I'm not aware of uses a function which would break if the ordering of parameters were changed. It might make more sense though to have the port argument follow the address argument, if you would prefer that?
